lang/python/python3/Makefile

```diff
@@ -7,15 +7,12 @@
 
 include $(TOPDIR)/rules.mk
 
-# The file included below defines PYTHON_VERSION
```

Comment:
fine to rename this;
fwiw: the reason i did it like this, is because i was diffing `python/Makefile` & `python3/Makefile`; so this kept the diff a bit smaller;
some things [at least in the beginning of the 2 Python packages] were common to both in some structure; so i would update 1 and diff the other to make sure i did not omit anything

now this is no longer needed;